.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,68 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  linux-test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install -g cmake-js@5.3.2
+    - run: npm install
+    - run: npx envinfo
+    - run: npm test
+      env:
+        CI: true       
+  windows_test:
+
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 13.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install -g cmake-js@5.3.2
+    - run: npm install
+    - run: npx envinfo
+    - run: npm test
+      env:
+        CI: true
+  mac_test:
+
+    runs-on: macos-latest
```

Comment:
since the build and test script is same. I think put the os on matrix will be much cleaner.

Reply:
I just fixed them, please review~

Reply:
What I meant is like this, this will remove the redundant setup.
```yml
    runs-on: ${{ matrix.operating-system }}
    strategy:
      matrix:
        operating-system: [ubuntu-latest, windows-latest, macos-latest]
```

Reply:
😁I understand what you mean. I just use your way to fix them